apps/builder/app/canvas/features/webstudio-component/selected-instance-connector.ts

```diff
@@ -192,19 +188,7 @@ export const SelectedInstanceConnector = ({
       unsubscribeScrollState();
       unsubscribeWindowResize();
       unsubscribeIsResizingCanvas();
-
-      // Retain selectedInstanceIsRenderedStore state if instanceSelector stays the same.
-      // Occasionally, an immediate call to selectedInstanceIsRenderedStore.set(true) may not update StylePanel state
-      // within the same batch as the current effect unsubscribe (e.g., due to delayed postMessage). (and cause to lost scroll position)
-      // This rare issue is difficult to reproduce, occurring roughly once every 100 calls.
-      if (
-        areInstanceSelectorsEqual(
-          selectedInstanceSelectorStore.get(),
-          instanceSelector
-        ) === false
-      ) {
-        selectedInstanceIsRenderedStore.set(false);
-      }
+      selectedInstanceIsRenderedStore.set(undefined);
```

Comment:
I would give a link to webstudio-component useEffect here

Reply:
Addded